pkg/cluster/k8sres.go

```diff
@@ -1169,10 +1171,25 @@ func (c *Cluster) generateCloneEnvironment(description *acidv1.CloneDescription)
 			})
 	} else {
 		// cloning with S3, find out the bucket to clone
+		cloneScope := cluster
+		cloneS3Bucket := c.OpConfig.WALES3Bucket
+		cloneSuffix := getBucketScopeSuffix(description.UID)
+
+		if description.S3WalPath != "" {
```

Comment:
don't we need to check that it is the valid S3 bucket name or at least log from where we are trying to clone to ease debugging?